File: A/arithmetic_array.c
#include <stdio.h>

void solve() {
    int n, a;
    scanf("%d", &n);
    int s = 0;
    for (int i = 0; i < n; i++) {
        scanf("%d", &a);
        s += a;
    }
    if (n == s) {
        printf("0\n");
    } else if (s < 0 || n > s) {
        printf("1\n");
    } else {
        printf("%d\n", s - n);
    }
}

int main(void) {
    int t;
    scanf("%d", &t);
    while (t--) {
        solve();
    }
    return 0;
}

File: A/arithmetic_array.py
import sys
input = sys.stdin.readline

def solve():
    n = int(input())
    a = list(map(int, input().split()))
    s = sum(a)
    m = 0 if n == s else 1 if (s < 0 or n > s) else s - n
    print(m)

for _ in range(int(input())):
    solve()
